### PR TITLE
[BD-46] docs: update README and fix feedback_template location

### PR DESCRIPTION
**Pull request by viktorrusakov**
## Description

Include a description of your changes here, along with a link to any relevant Jira tickets and/or Github issues.

### Deploy Preview

Include a direct link to your changes in this PR's deploy preview here (e.g., a specific component page).

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [ ] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [ ] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [ ] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @viktorrusakov!


When this pull request is ready, tag your edX technical lead.

File: README.md [hunk 1]
```diff
@@ -417,9 +416,9 @@ Paragon components can have different behavior in the MFE environment. `example`
 
 Steps to install the `example` app.
 
-1. `cd example && npm install` to install dependencies.
+1. `npm install` to install dependencies.
 2. Launch any devstack. It is required for MFE to login into the system and set up configs.
-3. `npm start` to start the app.
+3. `npm run start-example-mfe` to start the app.
```
Comment (adamstankiewicz) [inline README.md:421]: 🎉

File: README.md [hunk 2]
```diff
@@ -238,11 +238,10 @@ When developing a new component you should generally follow three rules:
 
 ##### 1. Start the documentation site development server
 
-The Paragon documentation site serves both as documentation and as a workbench to create your component within. To see your component in action, you need to run the documentation site locally. (Note you need to install dependencies both in the project root and the `www` directory)
+The Paragon documentation site serves both as documentation and as a workbench to create your component within. To see your component in action, you need to run the documentation site locally.
 
 ```
 npm install
-cd www
```
Comment (adamstankiewicz) [inline README.md:245]: Love that we no longer need to `cd www` in order to start the docs site locally now :)

**Review by adamstankiewicz — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (viktorrusakov) [reply README.md:245]: Yeah, that was definiteley annoying sometimes haha

**Comment by openedx-webhooks** [comment]
@viktorrusakov 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/2252&entry.1484655318=https://github.com/viktorrusakov&entry.752974735=2023-05-03+12:52&entry.1917517419=2023-05-03+13:26&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**